lighthouse-core/test/audits/errors-in-console-test.js

```diff
@@ -124,6 +124,22 @@ describe('ConsoleMessages error logs audit', () => {
       'TypeError: Cannot read property \'msie\' of undefined');
   });
 
+  // https://github.com/GoogleChrome/lighthouse/issues/10198
+  it('filters out blocked_by_client.inspector messages', () => {
+    const auditResult = ErrorLogsAudit.audit({
+      ConsoleMessages: [{
+        'source': 'exception',
+        'level': 'error',
+        'timestamp': 1506535813608.003,
+        'url': 'https://www.facebook.com/tr/',
+        'text': 'Failed to load resource: net::ERR_BLOCKED_BY_CLIENT.Inspector',
```

Comment:
IMO this test case is unnecessary, given the next few tests are exercising the filter feature.

Reply:
I would like to see the test remain as it describes a new, important part of this audit's default state of being. No real preference for position on my end :)

Reply:
it's not new, it's a parametrization of what multiple previous test cases has already proved to work (that the `ignoredPatterns` options works as intended). it might as well be `assert.equal(ErrorLogsAudit.defaultOptions().ignoredPatterns, ['whatever']);` (my preferred change). Just an idea- I like the process of tests building on top of each other.

At the least, please move the test to _after_ the ones that assert filtering works.

Reply:
> it might as well be assert.equal(ErrorLogsAudit.defaultOptions().ignoredPatterns, ['whatever']); (my preferred change, actually)

Sorry, I didn't realize that's what you were suggesting. It sounded to me like you wanted the assertion of this being the default removed, which I didn't :)

Reply:
It wasn't originally, your feedback evolved my stance :)

Reply:
Not sure why we're going to town on this fairly innocuous test, but I'll join in too :)

The test seems good as is to me. We don't care about how exactly the default pattern comes in, just that `ERR_BLOCKED_BY_CLIENT` errors are filtered out, so that's what should be tested.

If e.g. we decide we want to merge `ERR_BLOCKED_BY_CLIENT.Inspector` into whatever options are passed in instead of having the passed-in options overwrite the default (the [normal auditOptions behavior](https://github.com/GoogleChrome/lighthouse/blob/c3252522839c5eeba1efcc3a56c97bc0606f9d6c/lighthouse-core/runner.js#L346)) -- so `ERR_BLOCKED_BY_CLIENT.Inspector` couldn't go into `defaultOptions` -- there's no reason for this test to have to be rewritten.

Agreed that it makes sense to move into the `'options'` test block, and (supernit) maybe `it('filters out blocked_by_client.inspector messages by default', ...`?

Reply:
test left impl left as is. moved it to a new `defaultOptions` group right after `options`:

![image](https://user-images.githubusercontent.com/39191/104387183-6c8a9000-54eb-11eb-9501-976f9573c509.png)